lib/index.js

```diff
@@ -15,9 +16,15 @@ bedrock.events.on('bedrock.tests.run', state => {
   if(brTest.test.shouldRunFramework('karma')) {
     return run(state);
   }
+  if(brTest.test.shouldRunFramework('karma-coverage')) {
+    return run(state, {coverage: true});
+  }
+  if(brTest.test.shouldRunFramework('karma-coverage-ci')) {
```

Comment:
This is a little weird ... but I'm ok with it.

Reply:
It's seemed a reasonable way to do it given the setup we already had available and how the karma config bits work.  I'm open to other approaches but maybe this is a fine incremental improvement for now.